src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -930,7 +930,7 @@ const WorkflowView = _.flow(
           this.setState({ selectingData: false })
         },
         onSuccess: model => this.setState({ entitySelectionModel: model, selectingData: false }),
-        workspace,
+        workspace: ws,
```

Comment:
So before workspace was undefined because the wrong variable name was being referenced?

Reply:
The workspace details API returns a response like:
```
{ accessLevel: ..., workspace: { ... }, ... }
```

In WorkflowView's renderSummary function, `workspace` refers to that inner `workspace` object and `ws` to the whole thing.
https://github.com/DataBiosphere/terra-ui/blob/38b2a5535925ab2dd08e20cbe8b6b4c100228323/src/pages/workspaces/workspace/workflows/WorkflowView.js#L588

This changes the type of DataStepContent's `workspace` prop: https://github.com/DataBiosphere/terra-ui/pull/2886/files#diff-53608f0ffab957297bc01f1fd7a6bd48d53fc2315a86b575f1ebfc5d9fa0322bR20